pages/design/atomic-template-empty.md

```diff
@@ -0,0 +1,82 @@
+---
+name:
+test_type: atomic
+
+description: |
+  This rule checks ...
+
+success_criterion:
+- x.x.x
```

Comment:
Add "# (Name of Success Criterion)" here? We have that in most rules...
Applies to Composed rules as well